samples/hosting/nservicebus-host/Version_6/Sample/ConfigErrorQueue.cs

```diff
@@ -0,0 +1,13 @@
+﻿using NServiceBus.Config;
+using NServiceBus.Config.ConfigurationSource;
+
+class ConfigErrorQueue : IProvideConfiguration<MessageForwardingInCaseOfFaultConfig>
```

Comment:
use the code based api instead http://docs.particular.net/nservicebus/errors/#configure-your-error-queue-using-code